braid/relax.c

```diff
@@ -216,7 +215,8 @@ _braid_FCRelax(braid_Core  core,
                /* Note that we initialize StepStatus here in a non-standard
                 * way, and hence cannot use _braid_Step(...). */
                _braid_StepStatusInit(time_left, ta[ci-ilower], ci-cfactor-1, tol,
-                                     iter, level, nrefine, gupper, status);
+                                     iter, level, nrefine, gupper, 
+                                     braid_ASCaller_FCRelax, NULL, status);
```

Comment:
I'm just ignorant here -- can you explain why the basis argument is sometimes NULL (like here) and sometimes not NULL?